tests/prague/eip2537_bls_12_381_precompiles/test_bls12_g1mul.py

```diff
@@ -0,0 +1,240 @@
+"""
+abstract: Tests BLS12_G1MUL precompile of [EIP-2537: Precompile for BLS12-381 curve operations](https://eips.ethereum.org/EIPS/eip-2537)
+    Tests BLS12_G1MUL precompile of [EIP-2537: Precompile for BLS12-381 curve operations](https://eips.ethereum.org/EIPS/eip-2537).
+"""  # noqa: E501
+
+import pytest
+
+from ethereum_test_tools import Environment
+from ethereum_test_tools import Opcodes as Op
+from ethereum_test_tools import StateTestFiller, Transaction
+
+from .helpers import vectors_from_file
+from .spec import FORK, PointG1, Scalar, Spec, ref_spec_2537
+
+REFERENCE_SPEC_GIT_PATH = ref_spec_2537.git_path
+REFERENCE_SPEC_VERSION = ref_spec_2537.version
+
+pytestmark = [
+    pytest.mark.valid_from(str(FORK)),
```

Comment:
Personal preference here but I like using `"Prague"` over importing FORK from spec. Should shave some code but happy to keep like this

Reply:
Fixed! :+1: